shared/backend/aws/null_compute_backend.go

```diff
@@ -0,0 +1,73 @@
+package aws
```

Comment:
NullBackend is a dummy compute backend that is passed into Backend via launch type to avoid failure on creation, as k8s cluster has not been selected yet.